pages/juang961225/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>carrusel</title>
+    <link rel="stylesheet" href="../juang961225/style.css">
+</head>
+<body>
+<!--contenedor general-->
+    <main class="main">
+<!--titulo principal-->
+        <h2 class="main__title">word of warcraft class</h2>
```

Comment:
Sup buddy, I think it's better you keep the hierarchy of the headlines so this specifically talking it should be set with an H1 tag.

Reply:
solved